server/src/main/java/com/thoughtworks/go/server/service/BackupService.java

```diff
@@ -91,22 +104,29 @@ public BackupService(DataSource dataSource, ArtifactsDirHolder artifactsDirHolde
         this.timeProvider = timeProvider;
     }
 
+    void backupViaTimer(HttpLocalizedOperationResult result) {
+        performBackupWithoutAuthentication(Username.CRUISE_TIMER, result, BackupInitiator.TIMER);
+    }
+
     public ServerBackup startBackup(Username username, HttpLocalizedOperationResult result) {
         if (!goConfigService.isUserAdmin(username)) {
             result.forbidden("Unauthorized to initiate Go backup as you are not a Go administrator", HealthStateType.forbidden());
             return null;
         }
+        return performBackupWithoutAuthentication(username, result, BackupInitiator.USER);
+    }
+
+    private ServerBackup doPerformBackup(Username username, HttpLocalizedOperationResult result) {
         GoMailSender mailSender = goConfigService.getMailSender();
         synchronized (BACKUP_MUTEX) {
-            DateTime now = timeProvider.currentDateTime();
-            final File destDir = new File(backupLocation(), BACKUP + now.toString("YYYYMMdd-HHmmss"));
+            DateTime backupTime = timeProvider.currentDateTime();
+            final File destDir = new File(backupLocation(), BACKUP + backupTime.toString("YYYYMMdd-HHmmss"));
             if (!destDir.mkdirs()) {
-                result.badRequest("Failed to perform backup. Reason: Could not create the backup directory.");
+                result.internalServerError("Failed to perform backup. Reason: Could not create the backup directory.");
```

Comment:
The status code change was intentional. `422` seems to indicate a "client error", `500` feels more appropriate since it's clearly a server error.